src/core/core.defaults.js

```diff
@@ -20,7 +20,9 @@ defaults._set('global', {
 	defaultFontSize: 12,
 	defaultFontStyle: 'normal',
 	defaultLineHeight: 1.2,
-	showLines: true
+	showLines: true,
+	defaultStrokeStyle: '',
```

Comment:
Not sure about adding these to the global defaults. It may give the impression that they work everywhere.

Reply:
for me is hard to say... it is the first time that I send a PR to this project, so you will tell me if it make sense or not 😃

Reply:
Probably better to remove for now

Reply:
Oki! Removed from here. 
Also, following the idea that this is not a default feature, I moved the tick stuff to the `parseTickFontOptions`, I think that it makes more sense there. Thanks!